### PR TITLE
feature/OC-30 add condition for related inspection

### PR DESCRIPTION
**Pull request by LucioDarcon**
Signed-off-by: luciodarcon <luciiodarcon@gmail.com>

### CODE REVIEW

File: ui-library/src/main/java/br/com/stant/libraries/uilibrary/components/informationcardview/InformationCardView.java [hunk 1]
```diff
@@ -98,6 +103,15 @@ public void setArrowVisibility(Integer visibility) {
         mInformationCardViewBinding.informationCardViewArrowImageView.setVisibility(visibility);
     }
 
+    @Override
+    public void setServiceInspectionFormFilledId(Integer serviceInspectionFormFilledId) {
+        if(serviceInspectionFormFilledId != null) {
```
Comment (MaximoDouglas) [inline ui-library/src/main/java/br/com/stant/libraries/uilibrary/components/informationcardview/InformationCardView.java:108]: Add space after 'if'

File: ui-library/src/main/res/drawable/ic_warning.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="16dp"
+    android:height="16dp"
+    android:viewportWidth="512"
+    android:viewportHeight="512">
+
+    <path
+        android:fillColor="#55687E"
+        android:pathData="M500.974,365.071L327.473,64.556C312.758,39.075 285.371,23.245 256,23.245c-29.371,0 -56.758,15.83 -71.473,41.313L11.026,365.071C3.812,377.565 0,391.806 0,406.255c0,45.49 37.01,82.5 82.5,82.5h347c45.49,0 82.5,-37.01 82.5,-82.5C512,391.806 508.187,377.565 500.974,365.071zM429.5,473.755h-347c-37.22,0 -67.5,-30.28 -67.5,-67.5c0,-11.82 3.118,-23.468 9.017,-33.685L197.518,72.057C209.561,51.201 231.97,38.245 256,38.245c24.03,0 46.439,12.956 58.482,33.813l173.501,300.514c5.898,10.217 9.017,21.864 9.017,33.685C497,443.475 466.72,473.755 429.5,473.755z" />
+    <path
```
Comment (MaximoDouglas) [inline ui-library/src/main/res/drawable/ic_warning.xml:10]: Add enter before this line

File: ui-library/src/main/res/drawable/ic_warning.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        android:fillColor="#55687E"
+        android:pathData="M466.333,385.069l-173.5,-300.511C285.241,71.411 271.128,63.244 256,63.244s-29.241,8.166 -36.832,21.313l-79.251,137.267c-2.071,3.587 -0.842,8.174 2.745,10.245c3.588,2.069 8.174,0.841 10.245,-2.745l79.251,-137.267c4.92,-8.52 14.056,-13.813 23.842,-13.813s18.923,5.294 23.843,13.814l173.5,300.512c2.427,4.203 3.657,8.808 3.657,13.686c0,15.163 -12.337,27.5 -27.5,27.5h-347c-15.163,0 -27.5,-12.337 -27.5,-27.5c0,-4.878 1.23,-9.482 3.657,-13.686l79.25,-137.267c2.071,-3.587 0.842,-8.174 -2.745,-10.245c-3.59,-2.071 -8.175,-0.843 -10.245,2.745L45.667,385.07C41.906,391.583 40,398.71 40,406.255c0,23.435 19.065,42.5 42.5,42.5h347c23.435,0 42.5,-19.065 42.5,-42.5C472,398.71 470.094,391.582 466.333,385.069z" />
+    <path
```
Comment (MaximoDouglas) [inline ui-library/src/main/res/drawable/ic_warning.xml:13]: Add enter before this line

File: ui-library/src/main/res/drawable/ic_warning.xml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        android:fillColor="#55687E"
+        android:pathData="M256,128.755c-17.921,0 -32.5,14.579 -32.5,32.5v135c0,17.921 14.579,32.5 32.5,32.5s32.5,-14.579 32.5,-32.5v-135C288.5,143.334 273.921,128.755 256,128.755zM273.5,296.255c0,9.649 -7.851,17.5 -17.5,17.5s-17.5,-7.851 -17.5,-17.5v-135c0,-9.649 7.851,-17.5 17.5,-17.5s17.5,7.851 17.5,17.5V296.255z" />
+    <path
```
Comment (MaximoDouglas) [inline ui-library/src/main/res/drawable/ic_warning.xml:16]: Add enter before this line

Comment (LucioDarcon) [reply ui-library/src/main/res/drawable/ic_warning.xml:19]: Done!

File: ui-library/src/main/res/drawable/ic_warning.xml (hunk 3 above)
Comment (LucioDarcon) [reply ui-library/src/main/res/drawable/ic_warning.xml:15]: Done!

File: ui-library/src/main/res/drawable/ic_warning.xml (hunk 2 above)
Comment (LucioDarcon) [reply ui-library/src/main/res/drawable/ic_warning.xml:11]: Done!

File: ui-library/src/main/java/br/com/stant/libraries/uilibrary/components/informationcardview/InformationCardView.java (hunk 1 above)
Comment (LucioDarcon) [reply ui-library/src/main/java/br/com/stant/libraries/uilibrary/components/informationcardview/InformationCardView.java]: Done! @MaximoDouglas

**Review by MaximoDouglas — APPROVED** [review]

**Review by VilelaAgra — APPROVED** [review]

**Pull request merged**